automation_tools/__init__.py

```diff
@@ -1173,7 +1173,6 @@ def ak_install(admin_password=None, clean_beaker=True,
 
     # Clean up and install with basic packages.
     clean_rhsm()
-    update_basic_packages()
```

Comment:
We are facing this as DOGFOOD is not 100% in sync with CDN.

Anyways, I was answering to "system is not yet registered" here. :)